src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    static final int maxNumber = 100;
```

Comment:
Is there are any reason to make this variable package-private? Also please name this variable properly, according to naming convention